test_2_option.js

```diff
@@ -0,0 +1,118 @@
+const expect = require("expect");
+
+function greaterNum(num1, num2) {
```

Comment:
you should not copy functions here, instead you should require them from the corresponding file

Reply:
not addressed